core/audits/server-response-time.js

```diff
@@ -61,6 +68,14 @@ class ServerResponseTime extends Audit {
     const mainResource = await MainResource.request({devtoolsLog, URL: artifacts.URL}, context);
 
     const responseTime = ServerResponseTime.calculateResponseTime(mainResource);
+    if (responseTime === null) {
+      return {
+        score: 0,
+        runWarnings: [str_(UIStrings.missingTimingWarning)],
+        notApplicable: true,
+      };
+    }
```

Comment:
Maybe we could just throw an error here?

Reply:
Yeah throwing an error sounds good unless we are expecting this to happen in a number of cases